pkg/scheduler/capabilities/volumebinding/binder_test.go

```diff
@@ -2457,12 +2457,12 @@ func TestGetEligibleNodes(t *testing.T) {
 		eligibleNodes := testEnv.binder.GetEligibleNodes(logger, scenario.pvcs)
 
 		// Validate
-		if equality.Semantic.DeepEqual(scenario.eligibleNodes, eligibleNodes) {
+		if reflect.DeepEqual(scenario.eligibleNodes, eligibleNodes) {
```

Comment:
Is this changed back to reflect.DeepEqual?

Reply:
There was a deviation in conflict resolution and it has been fixed.